src/_playground/Routes.js

```diff
@@ -327,10 +342,20 @@ export class Routes extends Component {
             );
         });
 
+        const sideBarClasses = classnames({
+            'frDocs-Sidebar': this.state.showNav,
+            'frDocs-Sidebar__visibility': !this.state.showSideNav
+        });
+
+        const docsContentClasses = classnames({
+            'frDocs-Content': this.state.showNav,
```

Comment:
The base class `frDocs-Content` shouldn't be conditional otherwise you can lose all styling on that section.